__tests__/require-usememo.test.ts

```diff
@@ -327,6 +327,14 @@ describe('Rule - Require-usememo', () =>  {
           return {x, y};
         }`,
       },
+      // ignoredPropNames
+      {
```

Comment:
I think it'd be good to just add a few more cases, like using inline object/functions and const functions as well, just to be safe.

Reply:
@arthurgeron Thank you for your review. I add a few more case.